tensorrt_llm/_torch/modules/rotary_embedding.py

```diff
@@ -179,7 +179,7 @@ def forward(
                     k = k.view(seq_len, -1)
                     return [q, k]
 
-        if use_gptj_style_rope:
+        elif use_gptj_style_rope:
```

Comment:
This does not look right to me. If you can't run `flashinfer_apply_rope_inplace` with `interleave=True`, the computation is gonna be wrong.

Reply:
Resolved.